src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div
+      class="card"
+      data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED@UA/A)"
+        class="card__foto"
+      >
+
+      <div class="card__description">
+        <div class="card__description-product">
+          APPLE A1419 iMac 27&quot Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__description-code">
+          Product code: 195434
+        </div>
+      </div>
+
+      <div class="stars-viewed">
```

Comment:
should be BEM element of `card`

Reply:
fix it 
![image](https://user-images.githubusercontent.com/98764641/209652786-6319c43c-9edf-47b6-9c4c-24286d2edb53.png)

Reply:
i have no ideas, how i can fix it